tshark-4.4.yaml

```
@@ -45,6 +47,68 @@ pipeline:
   - uses: strip
 
 subpackages:
+  - name: ${{package.name}}-libs
+    dependencies:
+      provides:
+        - tshark-libs=${{package.full-version}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/lib
+          mv "${{targets.destdir}}"/usr/lib/* "${{targets.subpkgdir}}"/usr/lib
+    description: ${{package.name}} libraries
+    test:
+      pipeline:
+        - uses: test/tw/ldd-check
+
+  - name: editcap
+    description: Edit and/or translate the format of capture files
+    dependencies:
+      runtime:
+        - ${{package.name}}-libs
+    pipeline:
+      - runs: |
+          install -Dm755 "${{targets.destdir}}/usr/bin/editcap" "${{targets.contextdir}}/usr/bin/editcap"
```

```suggestion
          mkdir -p "${{targets.contextdir}}/usr/bin"
          mv "${{targets.destdir}}/usr/bin/editcap" "${{targets.contextdir}}/usr/bin/editcap"
```
`install` is just copying the file from one place to another, the tshark package should still have that binary in it if you dont `mv` instead

---

> We want to have editcap in the main package here.
I was thinking we could just add a runtime dependency for `editcap` on `tshark` instead (@kranurag7 )

---

> The customer wants to install editcap without installing the whole wireshark suite.

Yupyup! Splitting it out with `mv` lets us _not_ have `editcap` on `tshark`, then we can just depend on it on runtime then BAM we have the split out subpackage _and_ full tshark when we `apk add tshark`

---

I see what you mean
But, I think if we were to receive more requests like this in the future (for example: just dumpcap or capinfos), it would be more maintainable to just have a library subpackage like ${{package.name}}-libs.